.github/workflows/build.yml

```diff
@@ -43,14 +43,22 @@ jobs:
         uses: actions/cache@v2.1.5
         with:
           path: tools
-          key: ${{ runner.os }}-tools-${{ hashFiles('setup.cake') }}
+          key: ${{ runner.os }}-tools-${{ hashFiles('setup.cake') }}-${{ hashFiles('.config/dotnet-tools.json') }}
```

Comment:
The version of cake is the only dotnet-tool used, and it is explicitly set in this workflow, as such there is no need to change the cache key here.